pkgs/top-level/stage.nix

```diff
@@ -272,6 +272,7 @@ let
           if stdenv.isLinux
           then makeMuslParsedPlatform stdenv.hostPlatform.parsed
           else stdenv.hostPlatform.parsed;
+        gcc = lib.optionalAttrs (stdenv.hostPlatform ? gcc) stdenv.hostPlatform.gcc;
       } // lib.optionalAttrs (stdenv.hostPlatform.system == "powerpc64-linux") {
         gcc.abi = "elfv2";
```

Comment:
This will not merge correctly on ppc. Also, maybe we should inherit _all_ the values of the platform, and only change `isStatic` and `parsed`?

Reply:
> This will not merge correctly on ppc

Ups, I'll move the // to the gcc attribute instead.

> maybe we should inherit all the values of the platform

I don't know the consecuences of that, so I only wanted to do the minimal changes that fix the current problem. I assume for other pkgs* sets some similar should be done.